inspirehep/pidstore/minters/base.py

```diff
@@ -28,7 +30,10 @@ def validate(self):
             raise MissingSchema
 
     def get_pid_values(self):
-        return get_value(self.data, self.pid_value_path, default=None)
+        pid_values = get_value(self.data, self.pid_value_path, default=[])
+        if not isinstance(pid_values, (tuple, list)):
+            pid_values = force_list(pid_values)
+        return dedupe_list(pid_values)
```

Comment:
maybe simply
```suggestion
        return set(pid_values)
```
as the elements should be hashable, and the order doesn't matter?